app/views/paintings/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= @painting.name %>
+<% @painting.each do |painting| %>
+  <%= painting.title %><%= painting.type %>
```

Comment:
Maybe add a little space or dash between these two fields, and change .type to .category 👍